exporter/modules_test.go

```diff
@@ -8,7 +8,7 @@ import (
 	"github.com/prometheus/client_golang/prometheus"
 )
 
-func TestModules(t *testing.T) {
+func TestModulesv74(t *testing.T) {
```

Comment:
In theory I think these 2 test suits could be merged into one function with parameters for v7.4 and v8.0, but I'm not sure if it would be better.

Please let me know if it should be done.

Reply:
I think it's fine to keep these separate to make it more clear what version the tests are for.
My main objective is to keep the exporter itself simple and free from any `if redisVersion this else that` logic which we managed to keep up so far.